ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -71,7 +64,7 @@ tests:
     TESTS_TO_FOCUS=$(grep -E -o '\[crit\:high\]' tests/*_test.go | sort | uniq | sed -E 's/tests\/([a-z_]+)\.go\:.*/\1/' | tr '\n' '|' | sed 's/|$//')
     FUNC_TEST_ARGS='--ginkgo.noColor --ginkgo.focus='"$TESTS_TO_FOCUS"' --ginkgo.regexScansFilePath=true --junit-output='"$ARTIFACT_DIR"'/junit.functest.xml' \
         bash -x ./hack/functests.sh
-
+  cron: 2 3 * * *
```

Comment:
@danielBelenky I changed this now to start at 3AM, while the build job runs at 1AM.